pkg/limayaml/defaults.go

```diff
@@ -59,6 +78,16 @@ func FillDefault(y *LimaYAML, filePath string) {
 	if y.Containerd.User == nil {
 		y.Containerd.User = &[]bool{true}[0]
 	}
+	needsContainerdArchives := (y.Containerd.User != nil && *y.Containerd.User) || (y.Containerd.System != nil && *y.Containerd.System)
+	if needsContainerdArchives && len(y.Containerd.Archives) == 0 {
```

Comment:
Not sure why we need to check `needsContainerdArchives` at all; we could always set the default value, even if we know it won't be used.

Reply:
removed